core/new-gui/src/app/workspace/component/result-panel/debugger-frame/debugger-frame.component.html

```diff
@@ -3,7 +3,7 @@
   (click)="onClickSkipTuples()"
   *ngIf="breakpointTriggerInfo !== undefined"
   [disabled]="!breakpointAction"
-  [hidden] = true
+  [hidden]="true"
```

Comment:
just to make sure this change is correct, what's the difference between `=true` and `="true"`?

Reply:
there is no difference that I can tell, both are tested to be working. This change is originally introduced by the Prettier/ESLint.